### DISCUSSION TITLE
Task 1 Issues

### DISCUSSION THREAD

**Discussion by derekscholes** [USU-ITLS-x265-23-summer/class-repo#1] [Week 1 - Debug Requests]
I am unsure if anyone else has had this problem, but as a distance learner, I do not have a current USU ID, and being in a location far from Utah, I have been unable to sign up for Global Campus with my USU email. In addition it has only given me the option to take pictures of documentation and not upload documents from my computer. 

Another matter is the link to join the classroom. For me, it states when I click the classroom link that the "Page is not found". I am unsure if it is due to the link being incorrect or because I have not been able to join with my USU email. 
<img width="1787" alt="Screenshot 2023-05-07 at 4 00 08 PM" src="https://user-images.githubusercontent.com/132800749/236700090-9fa0fc07-822e-41e8-98a8-ba8e9a9c5dcf.png">
Has anyone else had this issue as well?
Are there any thoughts of what I can do to solve these problems for the first task?

**Comment by metasean** [comment]
@derekscholes - Can you access https://github.com/USU-ITLS-x265-23-summer/class-repo ?

**Reply by derekscholes** [reply]
I am able to access this link you have just posted and have access to all tasks and weekly assignments. It is only the specific link on this week 1 task page that does not function.

**Reply by metasean** [reply]
Can you access https://classroom.github.com/a/Qe0MErOh

**Reply by metasean** [reply]
@derekscholes - I'm going to go out on a limb and guess that you can access https://classroom.github.com/a/Qe0MErOh 

Assuming that you do have access to that link, click the green "Accept this assignment" in the lower right corner.  

The next page should inform you that your new repo is being created and that you need to refresh the page to see the new repo. (I'm not sure why GitHub doesn't do this refresh automatically, but they don't).  

Anyway, once you've _manually refreshed the page_ do you have this "Open in GitHub CodeSpaces" button on the resulting page?

<img width="400" alt="Screen Shot 2023-05-07 at 2 59 52 PM" src="https://user-images.githubusercontent.com/7552754/236703229-d4fb7a5a-7381-4c31-8b3a-9038f660ebf0.png">

If you do have the "Open in GitHub CodeSpaces", drop a response here to let me know, then click the button and work through the resulting reading and tasks.

If you do **_not_** have access to that button, I need to know **_asap_**.

**Reply by mpaxton16** [reply]
I had to do "Open in GitHub CodeSpaces". I was having the same issues, but I think that this worked.......just super unfamiliar with this so hoping I did it right.

**Reply by derekscholes** [reply]
It worked and I have worked through it. I don't know if I did it right either, as I couldn't do all the assignments through Codespaces but was able to edit on the provided link that is shown in blue.

**Reply by CraigButikofer** [reply]
I didn't have access at first but was able to click through until I got to the desired page.

**Comment by CraigButikofer** [comment]
My username is "CraigButikofer" I was having a tough time associating verifying my student status, and eventually I got an error message that said, "you seem to be having trouble verifying your student status, you'd better wait a couple days and try again."

**Reply by hardingjo** [reply]
I'm also having this issue of verifying my student status. My ID card doesn't have a date, or my transcripts dont match my username. Have you figured out how to do this?

**Reply by CJ3315** [reply]
I am also trying to figure out my student authentication, the first time I tried it was denied, do we need to use the student developer package? I think I am currently just using the free version.

**Reply by jakegros** [reply]
Yes, same here.  I even added the docs that a few helpful folks posted.  However, I suppose I tried too many times and need to wait a few days. 🤷‍♂️

**Comment by metasean** [comment]
@derekscholes , @mpaxton16 , & @CraigButikofer - 

Based on your comments and discussions, I've updated the directions, let me know if this helps 

https://github.com/USU-ITLS-x265-23-summer/class-repo/edit/main/Tasks/Task-3.md#notes-before-doing-following-assignments

**Reply by hardingjo** [reply]
I created an account after the previous two with issues, and things when smoothly for me-- everything seems good!

**Comment by hardingjo** [comment]
my username is 'hardingjo'

**Reply by metasean** [reply]
It appears that three things need to happen before I can fully you in the classroom.
- [x] you have to have a GitHub username  

- [ ] you have to have started one of the three GitHub assignments in [task 3](https://github.com/USU-ITLS-x265-23-summer/class-repo/blob/main/Tasks/Task-3.md)
   - [ ] Read everything through "Assignment 2" 
            (there are important steps that apply to assignments 2, 3, & 5)
   - [ ] start assignment 2 at https://classroom.github.com/a/Qe0MErOh 
   - [ ] follow the "NOTES BEFORE DOING FOLLOWING ASSIGNMENTS" for assignments 2, 3, & 5
   - [ ] one of the first steps for assignment 2, will result in a repo accessible at:
            https://github.com/USU-ITLS-x265-23-summer/github-starter-course-hardingjo
            
        *If you already have this repo, then you've probably made it private, which means that I don't have access to it, therefore I can't use it to connect your user account to the classroom roster.*  

        For example, you can see at derekscholes' entry at:
        https://github.com/USU-ITLS-x265-23-summer/github-starter-course-derekscholes
        yours should be https://github.com/USU-ITLS-x265-23-summer/github-starter-course-hardingjo  

        (you may also find it under your user account instead of the classroom account:
        https://github.com/derekscholes/skills-introduction-to-github
        in which case yours would be https://github.com/hardingjo/skills-introduction-to-github)
- [ ] Once the previous two things have been completed, then I can manually add you to the class roster.  FWIW: The fact that you left your message under the username 'hardingjo' confirms that you've successfully completed task 1 - assignment 0 - step 1 ✔️.

**Reply by CraigButikofer** [reply]
![image](https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/132943056/120b6fd3-b48c-46c7-95fe-54f81d95ae29)

I have tried several times to enroll in the Student GitHub but unsuccessful each time. One of the first attempts told me that my username should match my name as displayed on my student account so I changed my username to CraigButikofer, but ever since the emails show my initial username: cbutikof

Some of the links that I am clicking on result in this error:
![image](https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/132943056/e07569aa-22bf-434e-bb13-fac4dc777300)

![image](https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/132943056/de672450-1c5c-4cb4-ad1a-4f1177910e70)

I think I'm able to access the Repo, just not as a "student" and I'm not sure what I'm supposed to do on Assignment 2 (I believe I completed Assignment 1 by responding to and asking a question:

![image](https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/132943056/4796dc01-a635-48fa-b292-20a3c471cf99)

**Reply by CraigButikofer** [reply]
@metasean I was able to get the sign up as a student completed- Turns out the 15 time is the charm:
![image](https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/132943056/30e914fe-b853-491f-97f8-b4c5401e5d1a)
I have tried to join our classroom, but this is the error that I am getting:
![image](https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/132943056/9655c9f5-a4fd-41d2-8585-23b8e8725a3d)

**Reply by hardingjo** [reply]
@CraigButikofer what method of verification finally worked for you?

**Reply by CraigButikofer** [reply]
@hardingjo I had to allow my device to take a picture of me (just front facing- random picture of me), then I took a screenshot of my Banner and saved it as a PDF: 
![Craig Butikofer USU Summer'23](https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/132943056/3ca70958-183a-437b-bea2-0f551df38dcd)
Then I selected "Other...." and put a description of the student webpage or whatever was listed there.

**Reply by hardingjo** [reply]
@CraigButikofer But the prompt only allow you to upload one thing, and it has to be a picture. How did you upload your picture and a PDF?

**Reply by CraigButikofer** [reply]
@hardingjo The prompt only shows that one thing loads, but it will actually let you continue to upload files. It's pretty frustrating but it still worked for me. I just had to make sure that I did each step in the order that it was presented.

**Reply by coopsand** [reply]
I did something similar to this and it worked as well. I navigated to my schedule on Banner and submitted the following screenshot. This is the only image I uploaded, but I made sure my profile had my name and a description in it as well. If you are still struggling and the one above doesn't seem to be working, try this instead.

![Screenshot 2023-05-10 at 5 10 41 PM Large](https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133150498/2c27c29e-3fec-48e7-8a93-5c74989fbabe)

**Reply by chrisbabits** [reply]
Thanks for this suggestion, @coopsand!

**Comment by audreyhki** [comment]
My username is audreyhki
- hopefully, I am doing this right! It is a new experience for me.

**Reply by metasean** [reply]
The GitHub classroom is also a new experience for me.  
I know GitHub, but the classrooms are a different thing, but there are some benefits for y'all, so I'm trying to make it work.

**Comment by metasean** [comment]
@hardingjo  @audreyki -

1. will you please trying to sign up as a student - https://education.github.com/students
    (you may get some type of error, but do as much as you can until you get an error)
2. then try to join our classroom - [https://classroom.github.com/classrooms/131394480-itls-x265-23-summer>](https://classroom.github.com/classrooms/131394480-itls-x265-23-summer)
    (again, you make get some type of error, but do as much as you can)

**Comment by audreyhki** [comment]
Hello,

I believe I am in the classroom on GitHub and have been reviewing the course and the Tasks for this week. Originally when signing up, I did sign up as a student. Is it necessary for me to complete these two steps?
________________________________

**Reply by metasean** [reply]
Yes, please try these steps again.

**Reply by gabebott** [reply]
Yes, your are on the right track. Great job.

**Comment by Skye404** [comment]
Hi, I don't have any questions right now. I'm still reading through all the first week instructions, and it says to post here.

**Comment by hardingjo** [comment]
I've tried multiple methods of documentation for the student verification, but Github keeps denying with emails like this:
<img width="675" alt="Screenshot 2023-05-09 at 2 04 02 PM" src="https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/132959701/f12eb2c0-ce9a-42cb-a07f-81c163e22622">
Anyone that has gotten through this step, what did you do?

**Reply by ghost** [reply]
+1, I'm having the same issue

**Reply by audreyhki** [reply]
I have also received this exact message after every attempt.

**Comment by bst3ck** [comment]
Good Afternoon Everyone, I am excited to be part of this class. My GitHub username is bst3ck. It is a pleasure to meet you all virtually.

**Comment by metasean** [comment]
@CraigButikofer appears to found [a way to get verified](https://github.com/orgs/USU-ITLS-x265-23-summer/discussions/1#discussioncomment-5852509).

For those still having issues, have you tried the two upload (banner screenshot & camera snapshot) approach?

**Reply by hardingjo** [reply]
I have attempted this approach, but the prompt doesn't allow me to upload two things or to upload PDFs, only JPEG.

**Reply by CraigButikofer** [reply]
@hardingjo When I did it, there was no way to verify that the second document was attached. I just sort of had to assume that it was. The other issue that I had a few times was that I was trying to attach the same document. I think for verification reasons it was flagging that I had already attempted with that document. So when you start fresh:
1. allow your camera access
2. take a picture of yourself as the first document
3. upload a second document (a picture of your banner page-but make sure to save a new fresh one each time you attempt)

**Reply by CraigButikofer** [reply]
Another thing that I tried (not sure if it actually helped) was my original username was cbutikof but I thought there was an error message basically saying that the system needed to verify my username with the name that was showing up on my proof of verification. I did end up changing it to CraigButikofer, but a lot of the emails still said cbutikof..... but it worked in the end with the steps I listed above.... I hope that helps.

**Reply by metasean** [reply]
@hardingjo - From my end, it currently looks like you are connected. 👍

**Reply by hardingjo** [reply]
@metasean Does that mean I've been verified as a student? I also still cannot access the link for the classroom you gave: https://classroom.github.com/classrooms/131394480-itls-x265-23-summer%3E

**Reply by metasean** [reply]
@hardingjo  - There's an extra character on the end of that link - the link should be [https://classroom.github.com/classrooms/131394480-itls-x265-23-summer](https://classroom.github.com/classrooms/131394480-itls-x265-23-summer)  (without the `%3E` on the end of it)

Let me know if you can access it with that correction.

**Reply by metasean** [reply]
Also, where did you find the link with that extra character?

**Reply by hardingjo** [reply]
@metasean I still cannot access it.

**Reply by hardingjo** [reply]
@metasean I found that in an email thread, so I'm not sure where it is on the discussions here 
<img width="751" alt="Screenshot 2023-05-10 at 9 53 13 AM" src="https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/132959701/551ce14d-2a99-4ac3-9d77-310a1de41c99">
Looks like its correct on the email, so Im not sure where the extra characters came from. Still, the new link doesn't work.

**Reply by CraigButikofer** [reply]
@metasean I also could never access it from the email link, but I was able to access it from the canvas course if that is helpful.

**Comment by coopsand** [comment]
Hopefully I am doing this right! I haven't made my GitHub account a student account yet, but I believe that I have joined the correct classroom. My username is "coopsand" (Cooper Sanders, A02324659)

**Comment by aloeverah** [comment]
my user is 'aloeverah'. I'm Piper on Canvas. I'm struggling with the submission for a student account, none of my information is enough frustratingly enough. I keep playing around with it but for now can't figure out a solution.

**Comment by CJ3315** [comment]
Hello Everyone my Username is CJ3315! Excited to learn, this is all very new to me!

**Comment by bayleeriley** [comment]
Hey everyone! My username is bayleeriley

**Reply by chrisbabits** [reply]
Nice to see you in another course with me, Baylee! I believe you're also in video production this summer.

**Comment by gabebott** [comment]
Hi everyone,my username is gabebott. This is new for me but I'm excited to learn with everyone else this summer.

**Reply by gabebott** [reply]
I am currently a little lost on the navigation of github. I am just questioning if this is the right place to post for the assignment one question?

**Reply by chrisbabits** [reply]
Hi Gabe - From my understanding, it looks like this is the right place to post your question (and answer someone else's). I'm also new to GitHub. I'm hoping it'll become more intuitive in the coming days and weeks.

**Reply by jakegros** [reply]
It seems like it may work for a place to add Q&A for assignment 1 task 2.  I threw mine in with the Week 1 Discussions.  Not many there, though, so we will see.  I do have hope for navigation becoming easier.  I was completely lost (still wandering now) when I first tried to wrap my mind around this all.

**Comment by chrisbabits** [comment]
Hi everyone, my name is Chris. My username is chrisbabits. I'm a temporary assistant professor of History at Utah State University. I'm diversifying my skill set by enrolling in USU's ITLS MA program. I teach a good number of online courses, so I'm hoping to understand instructional tech better to give my students a better learning opportunity.

My questions have largely been answered by others in this discussion. I'm having the same issues confirming my status as a "Student," especially since I'm also faculty. Banner is currently down. But I'll be sure to get those materials uploaded and approved once Banner decides it wants to work again.

With that said, I'm curious -- what specific advantages might GitHub classroom hold over Canvas?

I imagine that it's better to have everything in one central location, especially since the kind of coding we'll be doing cannot be accomplished in Canvas. But I enjoy hearing the reasons that educators do what they do. That way, I can contemplate how I might change things up in my courses.

**Reply by chrisbabits** [reply]
I guess I have a question. It should be no shock that someone with a Ph.D. in History learns well from reading books. With that written, is there a book anyone would recommend for coding beginners? Has anyone read _The Pragmatic Programmer_, for instance?

**Comment by jakegros** [comment]
Hello everyone.  My username is jakegros
I am in the ITLS MS program.  I work at USU. I do a lot of document conversion with html so I am looking to grow my skillset.  I like to learn and dabble with new ideas.  GitHub and this course design is new to me. I am pretty stoked for this class.  As others have mentioned, it is a bit of a challenge to navigate this new structure.  However, I do enjoy learning by exploring and trying to figure things out.  I feel like this course will be robust and practical.

**Comment by beto0890** [comment]
Hello everyone !

I am in the same boat as all of you! so let godhead and stayed motivated. I am sure that by the end of this class, we will become pros at this! 
My username is beto0890
